### DISCUSSION TITLE
Food scraps management

### DISCUSSION THREAD

**Discussion by michaelhertel** [TandoorRecipes/recipes#1726] [Ideas]
Hi,

I have an idea for a feature request and would like to start the discussion about your interest. 
If the meal is cooked many times, we have Food scraps that we put into the freezer. To manage this we use a simple list. My idea is to have a similar list like the shopping list to manage the Food scraps. 
From this list you can put froozen meals on the meal plan if you have enough in your freezer.

**Comment by vabene1111** [comment]
Thanks for your suggestion. This is actually planned as some kind of inventory system. You will be able to have a set amount of any food (and recipes can be foods as well).

For this to properly work and get the most out of it we will need to implement unit conversion first but after that this can be done.
